doc/HOWTO_DOCUMENT.rst.txt

```diff
@@ -415,6 +415,14 @@ The sections of the docstring are:
       >>> np.add([1, 2], [3, 4])
       array([4, 6])
 
+    An instruction may be split across multiple lines, with each line after
```

Comment:
"instruction" seems like a weird description. Perhaps "An expression", "A statement", or even just "The example code"

Reply:
Indeed. Thanks.